.github/workflows/update_versions.yml

```diff
@@ -0,0 +1,47 @@
+# This is a basic workflow that is manually triggered
+
+name: Update Versions
+
+# Controls when the action will run. Workflow runs when manually triggered using the UI
+# or API.
+on:
+  workflow_dispatch:
+    # Inputs the workflow accepts.
+    inputs:
+      version:
+        description: 'New version to use for the extension. Example: 1.5.2'
+        required: true
+
+      branch:
+        description: 'Branch to be used when updating versions'
+        required: true
+
+      core-dependency:
+        description: 'If a version is provided, update AEPCore dependency in podspec and Package.swift'
+        required: false
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  update-versions:
+    runs-on: macos-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+
+    - name: Checkout
+      uses: actions/checkout@v3.1.0
+      with:
+        ref: ${{ github.event.inputs.branch }}
+
+    - name: Update AEPAudience (with core dependency update)
+      run: (sh ./Script/update-versions.sh -n Audience -v ${{ github.event.inputs.version }} -d "AEPCore ${{ github.event.inputs.core-dependency }}")
```

Comment:
You'll need to add AEPIdentity and AEPServices here as they are listed in the Podspec. For Services, you can either set the version same as Core or add a separate input above.